spec_helper.rb

```diff
@@ -39,6 +39,8 @@ module SpecHelper
     # This was a request that was added with Ruby 2.4.0
     allow(Fastlane::FastlaneRequire).to receive(:install_gem_if_needed).and_return(nil)
 
+    ENV['FASTLANE_PLATFORM_NAME'] = nil
```

Comment:
What's this for?

Reply:
some env var state that needed to be cleared out :(